uber/site_sections/preregistration.py

```diff
@@ -191,12 +191,31 @@ def banned(self, id):
         attendee, group = self._get_unsaved(id)
         return {'attendee': attendee}
 
+    def freereg_payment(self, session):
+        charge = Charge(listify(self.unpaid_preregs.values()))
+        if charge.total_cost == 0:
```

Comment:
For safety, let's have this as `charge.total_cost <= 0`, just in case we have an issue down the road with negative charge values for some reason.